udemy_enroller_chrome.py

```diff
@@ -19,9 +21,30 @@
 with open('settings.yaml') as f:    
     settings = yaml.load(f)
 
-email, password, zipcode = settings['udemy']['email'], settings['udemy']['password'], settings['udemy']['zipcode']
-
-driver = webdriver.Chrome(ChromeDriverManager().install())
+email, password, zipcode = (
+    os.environ.get("UDEMY_EMAIL", settings["udemy"]["email"]),
+    os.environ.get(
+        "UDEMY_PASSWORD",
+        settings["udemy"]["password"],
+    ),
+    settings["udemy"]["zipcode"],
+)
+
+is_ci_build = strtobool(os.environ.get("CI"))
+chrome_options = None
+if is_ci_build:
+    from selenium.webdriver.chrome.options import Options
+
+    # Having the user-agent with Headless param was always leading to robot check
+    user_agent = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/85.0.4183.102 " \
```

Comment:
Ye if I comment out the user-agent being added to chrome options above I always get the robot check. If I then check with the user-agent set as the one above it can get to the site no problem.

This is what it looks like without the above user-agent being injected: 
`Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/85.0.4183.102 Safari/537.36`

I'm not sure on limits of this at the moment but I'm able to rerun multiple times with no robot checks